### DISCUSSION TITLE
DirectX12 Support in the future

### DISCUSSION THREAD

**Discussion by ActiveXSturcture** [EQMG/Acid#115] [General]
Will be DirectX12 Graphic API add to the project?What Window framework will be used?Because as i known the DirectX12 only support WinForm,it not support GLFW.What abstract class will you achieve this API?

**Comment by bv94** [comment]
as far as i know
vulkan usually performs better than dx12
:>

**Comment by mattparks** [comment]
You can use DirectX12 and GLFW together: https://github.com/glfw/glfw/issues/1755#issuecomment-803786613

Acid currently has no plans to implement DirectX12 as a second graphics API; I was hoping for a project like [MoltenVK](https://github.com/KhronosGroup/MoltenVK) to be available in the future to support DirectX12 for UWP/XBox applications.
